bin/sdk/gutenberg.js

```diff
@@ -0,0 +1,56 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+const chalk = require( 'chalk' );
+const path = require( 'path' );
+const webpack = require( 'webpack' );
+
+const __rootDir = path.resolve( __dirname, '../../' );
+const baseConfig = require( path.join( __rootDir, 'webpack.config.js' ) );
+
+exports.compile = args => {
+
+	const options = {
+		outputDir: path.join( path.dirname( args.editorScript ), 'build' ),
+		...args
+	};
+
+	const config = {
+		...baseConfig,
+		...{
+			context: __rootDir,
+			mode: options.mode,
+			entry: {
+				'editor-script': options.editorScript,
+			},
+			externals: {
+				...baseConfig.externals,
+				wp: 'wp',
+			},
+			optimization: {
+				splitChunks: false,
+			},
+			output: {
+				path: options.outputDir,
+				filename: `[name].js`,
+				libraryTarget: 'window',
+				library: `gutenberg-extension-[name]`,
```

Comment:
It's a bit unclear if we actually need this with `libraryTarget: 'window'`?

https://webpack.js.org/guides/author-libraries/

Reply:
I just tested without it and hello-dolly worked just fine 🤷‍♂️ 

It would matter more if we make `libraryTarget` configurable

Reply:
Oh and _with_ `library`, we really need to ensure they're unique. Otherwise different files can end up overwriting each other:
```js
window['gutenberg-extension-editor-script']=function
```

So I'd drop it out to keep things more simple for now.